lms/djangoapps/courseware/views/views.py

```diff
@@ -2317,6 +2317,13 @@ def courseware_mfe_search_enabled(request, course_id=None):
     else:
         enabled = True
 
+    inclusion_date = settings.FEATURES.get('COURSEWARE_SEARCH_INCLUSION_DATE')
+    start_date = CourseOverview.get_from_id(course_key).start
+
+    # only include courses that have a start date later than the setting-defined inclusion date
+    if inclusion_date:
```

Comment:
Should we set `enabled` to `False` without the inclusion_date value?

Reply:
If there's no inclusion date, we should just rely on the default behavior in this view, which is why this is separated into a separate conditional.